sqlglot/expressions.py

```diff
@@ -5707,7 +5707,14 @@ class UnixToStr(Func):
 # https://prestodb.io/docs/current/functions/datetime.html
 # presto has weird zone/hours/minutes
 class UnixToTime(Func):
-    arg_types = {"this": True, "scale": False, "zone": False, "hours": False, "minutes": False}
+    arg_types = {
+        "this": True,
+        "format": False,
```

Comment:
Let's make `format` the last key in the dictionary to ensure there are no weird argument order shenanigans. For example if someone used `from_arg_list` and expected the second one to be `scale`, this change here would mess up the AST.